src/components/screens/OpeningTimesCard.js

```diff
@@ -0,0 +1,111 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { View, StyleSheet } from 'react-native';
+import styled from 'styled-components';
+
+import { normalize, colors } from '../../config';
+import { momentFormat } from '../../helpers';
+import { BoldText, RegularText } from '../Text';
+import { Wrapper, WrapperRow } from '../Wrapper';
+
+const TimeBox = styled.View`
+  flex: 1;
+  flex-direction: row;
+  flex-wrap: wrap;
+  margin-bottom: 5;
+`;
+
+const DateBox = styled(TimeBox)`
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+`;
+/* eslint-disable complexity */
+/* TODO: refactoring? */
+export const OpeningTimesCard = ({ openingHours }) => (
+  <Wrapper>
+    {!!openingHours &&
+      openingHours.map((item, index) => {
+        const { weekday, timeFrom, timeTo, dateFrom, dateTo, description } = item;
+        const returnFormat = 'HH:mm';
+        const returnFormatDate = 'DD.MM.YYYY';
+
+        return weekday ? (
+          <View key={index} style={styles.divider}>
+            <View>
+              <BoldText style={{ marginBottom: normalize(3) }}>{weekday}</BoldText>
+            </View>
+            <WrapperRow style={{ justifyContent: 'space-between' }}>
+              {(!!timeFrom || !!timeTo) && (
+                <TimeBox>
+                  {!!timeFrom && (
+                    <RegularText>{momentFormat(timeFrom, returnFormat)} -</RegularText>
```

Comment:
the ` -` is now always present, when there is a `timeFrom`, even if there is no `timeTo`. we could think of rendering it only if both are present?

Reply:
` {(!!timeFrom || !!timeTo) && <RegularText> -</RegularText>}`
 like this ?

Reply:
this means one OR the other...with `&&` you achieve one AND the other

Reply:
it is still `||`?!